lib/lfrfid/protocols/lfrfid_protocols.h

```diff
@@ -0,0 +1,32 @@
+#pragma once
+#include <toolbox/protocols/protocol.h>
+#include "../tools/t5577.h"
+
+typedef enum {
+    LFRFIDFeatureASK = 1 << 0, /** ASK Demodulation */
+    LFRFIDFeaturePSK = 1 << 1, /** PSK Demodulation */
+} LFRFIDFeature;
+
+typedef enum {
+    LFRFIDProtocolEM4100,
+    LFRFIDProtocolH10301,
+    LFRFIDProtocolIndala26,
+    LFRFIDProtocolIOProxXSF,
+    LFRFIDProtocolAwid,
+    LFRFIDProtocolFDXB,
+
+    LFRFIDProtocolMax,
+} LFRFIDProtocol;
+
+extern const ProtocolBase* lfrfid_protocols[];
+
+typedef enum {
+    LFRFIDWriteTypeT5577,
+} LFRFIDWriteType;
+
+typedef struct {
+    LFRFIDWriteType write_type;
+    union {
```

Comment:
union?

Reply:
To make it easier to extend the code later. We want not only T5577.